### DISCUSSION TITLE
Getting unknown for a few sensors

### DISCUSSION THREAD

**Discussion by djdarlo** [nathanmarlor/foxess_modbus#472] [Support] [answered]
I’ve recently installed the integration for my friend and a few of the sensors are showing unknown (shown in screenshot). Any idea why this may be?
![IMG_6895](https://github.com/nathanmarlor/foxess_modbus/assets/20640104/d9da2249-9293-4baf-b222-a8276dadcac7)

**Comment by canton7** [comment, marked as answer]
Those are integration sensors. They'll recover when they've seen their input sensors change a few times.

**Comment by djdarlo** [comment]
Gotcha.

**Comment by djdarlo** [comment]
Refreshed and working this morning. Thank-you.

**Reply by canton7** [reply]
Glad to hear!

**Comment by guba91** [comment]
> Those are integration sensors. They'll recover when they've seen their input sensors change a few times. 

Put this in the wiki! 😄 Thanks!

**Reply by canton7** [reply]
It's been in the wiki for months 😉 https://github.com/nathanmarlor/foxess_modbus/wiki/FAQs

**Reply by guba91** [reply]
> It's been in the wiki for months 😉 https://github.com/nathanmarlor/foxess_modbus/wiki/FAQs

No wayyyy hahahah sorry
